### ISSUE TITLE
Unable to compile node-sass build (Webpack/Parcel)

### ISSUE THREAD

**Issue by devanandb** [uikit/uikit#3833]
### UIkit version
3.1.0

### Issue
Unable to compile SASS from the src includes, both with Webpack and Parcel. The compilation is failing on these lines:
```
Invalid CSS after "...ng-primary when": expected selector, was "($deprecated = true"
Error: Invalid CSS after "...ng-primary when": expected selector, was "($deprecated = true"
        on line 125 of node_modules/uikit/src/scss/components/heading.scss
        from line 8 of node_modules/uikit/src/scss/components/_import.scss
        from line 7 of node_modules/uikit/src/scss/uikit-theme.scss
        from line 22 of resources/assets/styles/app.scss
>> .uk-heading-primary when ($deprecated = true) {

   -------------------------^
```
Looks like the deprecated flag is not added to the sass src, which looks like was recently added: "Add deprecated flag to Less variables" (from the changelog)

**Comment by mohamedhamad** [comment]
having the same issue. not sure how to resolve

**Comment by dimd13** [comment]
+1 same issue
`.uk-heading-primary when ($deprecated = true)`
It looks like a less directive.

**Comment by websam101** [comment]
+1

**Comment by mikkokam** [comment]
+1 under Angular-CLI

**Comment by rolldevio** [comment]
+1 same here

**Comment by dalu** [comment]
great, just when I was about to use something other than twbs

**Comment by dalu** [comment]
duplicate of #3832

**Comment by janschoenherr** [comment]
Thanks, this should be resolved with 3.1.1
